scripts/setup.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-export PATH=env/bin:${PATH}
+export PATH=$(poetry show -v | grep virtualenv | sed 's/Using virtualenv: //'):${PATH}
 
 python3 -m venv env
```

Comment:
Most of this stuff should be able to be replaced with `poetry install` etc.

I'd be interested to see if Poetry has a solution to the conditional install of `black`

Reply:
Looks like there is! https://github.com/sdispater/poetry/blob/master/pyproject.toml#L40 :tada:

Reply:
> Most of this stuff should be able to be replaced with `poetry install` etc.

I think that `poetry install` is entirely reasonable to remember.

https://j5.readthedocs.io/en/master/development/getting-started.html#setting-up is how we handle it for `j5`. It's literally just `poetry install`

Reply:
Yeah I think I agree. If you get the `black` install working, i'd be ok removing this script!

Reply:
c8b01cc